src/main/java/transforms/GeographyPointTransform.java

```diff
@@ -51,7 +51,20 @@ public Object apply( Map<String, String> row ) {
         lon = Parsers.parseDouble( lon );
 
         if (  lat != null && lon != null  ) {
```

Comment:
You can actually invert all this and lose the `else` and the code becomes more readable
```
if ( lat == null || lon == null ) { 
    return null; 
}
// processing code
```